components/footer/footer.html

```diff
@@ -0,0 +1,94 @@
+<footer>
+    <div class="container-fluid">
+      <div class="row subscription">
+        <div
+          class="col-12 col-lg-6 subscription--box d-flex flex-column justify-content-center align-items-center"
+        >
+          <h2 class="d-flex flex-column subscription--box--title">
+            <span>Newsletter</span>
+            <span>Subscription</span>
+          </h2>
+          <form class="subscription--box--email">
+            <div class="input-group mb-3">
+              <input
+                type="text"
+                placeholder="shovon.khan0099@gmail.com"
+                aria-label="Recipient's username"
+                aria-describedby="basic-addon2"
+                class="subscription--box--email--input position-relative"
+              />
+              <div class="input-group-append">
+                <button
+                  type="submit"
+                  class="btn btn-dark input-group-text subscription--box--email--btn position-absolute top-0 end-0"
+                  id="basic-addon2"
+                >
+                  &#8599;
+                </button>
+              </div>
+            </div>
+          </form>
+        </div>
+        <div
+          class="col-12 col-lg-6 d-flex align-items-center justify-content-center"
+        >
+          <img
+            class="subscription--image"
+            src="./assets/images/main/american-football-player.png"
+          />
+        </div>
+      </div>
+      <div
+        class="w-100 d-flex flex-column gap-sm-0 gap-5 flex-sm-row justify-content-sm-between flex-wrap align-items-center social-bar"
+      >
+        <div class="d-flex social-bar--list">
+          <div
+            class="social-bar--list--icon-container d-flex justify-content-center align-items-center"
+          >
+            <i
+              class="fa fa-facebook rounded-2 social-bar--list--icon-container--icon"
+            ></i>
+          </div>
+          <div
+            class="social-bar--list--icon-container d-flex justify-content-center align-items-center"
+          >
```

Comment:
![cf](https://github.com/Abdelhakam11/news/assets/106445705/17c436b0-a270-4cf8-81bd-77188d355bae)
1- padding top=48px  and padding-bottom =82px   not 42px and 42px
2- font size in icon social not same in design

Reply:
@hassanosamaa done.